src/main/java/com/google/api/codegen/config/PackageMetadataConfig.java

```diff
@@ -59,6 +61,9 @@
 
   protected abstract Map<TargetLanguage, Map<String, VersionBound>> protoPackageDependencies();
 
+  @Nullable
+  protected abstract Map<TargetLanguage, Map<String, VersionBound>> protoPackageTestDependencies();
```

Comment:
Note: I have added this protoPackageTestDependencies field to specify test dependencies for GAPIC library (specifically for Pubsub since the pubsub tests depends on the IAM gRPC package)